ChangeLog.md

```diff
@@ -20,6 +20,13 @@ See docs/process.md for more on how version tagging works.
 
 3.1.26 (in development)
 -----------------------
+- Inline with the recent changes to llvm and binaryen, emscripten will now, by
+  default, enable the sign-extension and mutable-globals WebAssembly proposals.
+  In order to do so the default minimum safari version (`MIN_SAFARI_VERSION`)
+  was updated from 12.0 to 14.1, and support for the old EdgeHTML engine
+  (`MIN_EDGE_VERSION`) was removed by default.  If you want to continue to
+  support older engines you can use these settings to specify alternative minium
+  versions which will result in the new proposals being disabled. (#17690)
```

Comment:
Maybe specify the actual flags, `-sMIN_SAFARI_VERSION=120000`, `-sMIN_EDGE_VERSION=44`, for those that want the old behavior, so they don't need to figure those out?